### PR TITLE
[GLUTEN-3625][FOLLOWUP] Fix IDEA load module issue

### PR DESCRIPTION
**Pull request by ulysses-you**
## What changes were proposed in this pull request?

Fix IDEA load module issue

## How was this patch tested?

manually test

### CODE REVIEW

**Comment by ulysses-you** [comment]
I always enable all the Spark profiles, so did not realize the idea issue..  @zzcclp can you try this pr ? thank you

**Comment by PHILO-HE** [comment]
cc @baibaichen

File: gluten-ut/pom.xml [hunk 1]
```diff
@@ -24,6 +24,11 @@
     <relativePath>../pom.xml</relativePath>
   </parent>
 
+  <modules>
+    <module>common</module>
+    <module>${spark.test.module}</module>
+  </modules>
```
Comment (PHILO-HE) [inline gluten-ut/pom.xml:30]: Does it work if just adding the below in shims/pom.xml,  similar to the above? 

```
  <modules>
    <module>common</module>
    <module>${spark.shim.module}</module>
  </modules>
```

**Comment by zzcclp** [comment]
manually test, it works for me in IDEA.

File: gluten-ut/pom.xml (hunk 1 above)
Comment (ulysses-you) [reply gluten-ut/pom.xml:30]: addessed and manually verified

**Review by PHILO-HE — APPROVED** [review]
Thanks for your quick response!
Let's wait for @baibaichen to verify. @zzcclp, please re-confirm the latest changes. Thanks!

**Comment by baibaichen** [comment]
it's ok  now.  let's merge!

**Pull request merged**